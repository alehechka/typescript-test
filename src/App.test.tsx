import React from 'react';
import ProjectSummary from './components/projects/ProjectSummary';
import renderer from 'react-test-renderer';

test('Project summary renders', () => {
	const component = renderer.create(<ProjectSummary project={{ title: '', content: '' }} />);
	let tree = component.toJSON();
	expect(tree).toMatchObject;
});
