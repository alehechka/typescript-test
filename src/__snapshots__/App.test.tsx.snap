// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Project summary renders 1`] = `
<div
  className="card z-depth-0 project-summary"
>
  <div
    className="card-content grey-text text-darken-3"
  >
    <span
      className="card-title"
    >
      
    </span>
    <p>
      Posted by 
       
    </p>
    <p
      className="grey-text"
    >
      Today at 10:17 PM
    </p>
  </div>
</div>
`;
